libs/dh/admin/feature-user-management/src/lib/tabs/users-tab/dh-users-tab.component.spec.ts

```diff
@@ -120,27 +125,29 @@ describe(DhUsersTabComponent, () => {
     const { store } = await setup();
 
     const inputValue = 'test';
-    const searchInput = screen.getByRole('textbox');
+    const searchInput = screen.getByRole('searchbox');
 
     userEvent.type(searchInput, inputValue);
-    tick(searchDebounceTimeMs);
+    tick(250);
```

Comment:
Question: This is now a hardcoded value, is this correct?

Reply:
It is correct, but also intentional (for now). That is because `searchDebounceTimeMs` was imported from `ui-search` library, which is no longer used in the `dh-users-tab` component. The idea is to expose `searchDebounceTimeMs` from `ui-util` library in a follow up PR.